### ISSUE TITLE
training results

### ISSUE THREAD

**Issue by Jaehun-Kim22** [Jaehun-Kim22/SCIC_Project#33]

**Comment by Jaehun-Kim22** [comment]
# batch size 16

<img width="386" alt="Screen Shot 2021-08-17 at 11 18 31 PM" src="https://user-images.githubusercontent.com/66521857/129742969-ee3e216f-7491-4425-ae81-938eced78f51.png">

**Comment by michael960725** [comment]
sent: 5e-06, class: 5e-05
![image](https://user-images.githubusercontent.com/86643680/129746734-d0368958-dbb5-43da-91b6-52b9ad166c6b.png)

**Comment by michael960725** [comment]
sent: 5e-06, class: 5e-05
![image](https://user-images.githubusercontent.com/86643680/129751226-25615445-3672-446b-b945-413523cfbb6a.png)

**Comment by michael960725** [comment]
sent: 1e-06 class: 3e-05
![image](https://user-images.githubusercontent.com/86643680/129755644-75b1c888-87a1-44e1-b41a-ee8ddda9c5ed.png)

**Comment by michael960725** [comment]
sent: 3e-06 class: 3e-05
![image](https://user-images.githubusercontent.com/86643680/129762866-a8dc2e7c-9162-4c14-aecd-14d4eabe5a7d.png)

**Comment by Jaehun-Kim22** [comment]
# batch size 32

<img width="367" alt="Screen Shot 2021-08-18 at 1 41 53 AM" src="https://user-images.githubusercontent.com/66521857/129766338-e62fdff6-8147-4f12-91c8-d21d7b5ecc21.png">

**Comment by michael960725** [comment]
sent: 7e-06 class: 3e-05
![image](https://user-images.githubusercontent.com/86643680/129769357-cd29e3af-7d89-44c4-b0c2-765a3d87bf69.png)

**Comment by michael960725** [comment]
sent: 3e-06 class: 3e-05
![image](https://user-images.githubusercontent.com/86643680/129787866-d9e4e33b-355b-4ad3-93a8-194f734369cb.png)

**Comment by autumnluv** [comment]
[v1(augment포함).xlsx](https://github.com/Jaehun-Kim22/SCIC_Project/files/7004677/v1.augment.xlsx)
[v2(augment포함).xlsx](https://github.com/Jaehun-Kim22/SCIC_Project/files/7004678/v2.augment.xlsx)
[v3(augment포함).xlsx](https://github.com/Jaehun-Kim22/SCIC_Project/files/7004679/v3.augment.xlsx)

**Comment by michael960725** [comment]
sent 3e-06 class 5e-05 v1-aug
![image](https://user-images.githubusercontent.com/86643680/129842465-99492408-899a-4b0e-8bfa-4137b3a81015.png)

**Comment by michael960725** [comment]
sent: 3e-06 class: 7e-05 v3-aug
![image](https://user-images.githubusercontent.com/86643680/129851825-40dea60e-f828-43d6-9c6a-18d52bccf046.png)

**Comment by michael960725** [comment]
sent: 6e-06 class: 5e-05
![image](https://user-images.githubusercontent.com/86643680/129858192-4a900a1e-8bad-4ddb-91df-7f5616d91d1e.png)

**Comment by michael960725** [comment]
sent: 3e-06 class: 3e-05 v3
![image](https://user-images.githubusercontent.com/86643680/129860120-8bc2e443-0c7d-4107-96a4-40c3c5ae07ba.png)

**Comment by michael960725** [comment]
sent: 3e-05 class: 4e-06 warm up: 0.2, weight decay: 0.1, 0.2
![image](https://user-images.githubusercontent.com/86643680/129889195-7f448c84-27ea-465f-a15a-fb9015cc3bb7.png)

**Comment by michael960725** [comment]
sent: 5e-06 class: 3e-05 warmup: 0.15 wd_1: 0.1, wd_2: 0.2
![image](https://user-images.githubusercontent.com/86643680/129914140-f0d53678-4a9c-400d-a792-9f6022497819.png)
